libs/galley-types/src/Galley/Types.hs

```diff
@@ -313,10 +313,10 @@ data OtrFilterMissing
     OtrReportAllMissing
   | -- | Complain only about missing
     --      recipients who are /not/ on this list
-    OtrIgnoreMissing (Set UserId)
+    OtrIgnoreMissing (Set UserId) -- TODO (mheinzel)
```

Comment:
TODO what?

```suggestion
    OtrIgnoreMissing (Set UserId) -- TODO (mheinzel): use 'OpaqueUserId'
```

Reply:
You must be looking at an old version of the code, this TODO has been done in `4ee3cdd`.